WalletWasabi.Gui/Global.cs

```diff
@@ -711,13 +727,8 @@ public async Task DisposeAsync()
 					}
 				}
 
-				Logger.LogDebug($"Step: {nameof(TorManager)}.", nameof(Global));
-
-				if (TorManager is { } torManager)
-				{
-					await torManager.StopAsync(Config.TerminateTorOnExit).ConfigureAwait(false);
-					Logger.LogInfo($"{nameof(TorManager)} is stopped.");
-				}
+				Logger.LogDebug("Dispose DI container.");
+				await Container.DisposeAsync().ConfigureAwait(false);
```

Comment:
This will dispose everything in the container that is IDisposable or IDisposableAsync?

Reply:
Yes, AFAIK this is how it behaves.